RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -769,6 +765,18 @@ void finalizeMaximize(boolean speculative)
 		}
 	}
 
+	if(auto_haveCCSC())
+	{
+		if(auto_handleCCSC())
+		{
+			addBonusToMaximize($item[Candy Cane Sword Cane], 1000); // we really want the CCSC if we can use it in a zone for a choice
```

Comment:
```suggestion
			autoEquip($item[Candy Cane Sword Cane]); // we really want the CCSC if we can use it in a zone for a choice
```
Just force the maximizer to equip it when we want it. Messing with bonuses just creates problems for the future.

Reply:
Yes it does cause problems iin the present too. I had 1000 as the bonus to force it above June cleaver, but in my most recent run I ran into, for the first time, cleaver being equipped when we had the 1000bonus for the CCSC so I added code to pre_adv to force equip CCSC if auto_handleCCSC is true so the maximizer statement is duplicative. It's fully removed now and hopefully once Mafia recognizes the boosts that the CCSC gets when its skills are used, it will actually be used anyway.

Reply:
Yeah the problem with things like the Cleaver are it getting better the more you use it so the maximizer weighting is going to continually lean towards it the longer your day goes.